File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio04.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio04 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        System.out.println("Insira um número inteiro: ");
        int a = entrada.nextInt();
        System.out.println("Insira outro número inteiro: ");
        int b = entrada.nextInt();

        int soma = a + b;
        System.out.println("A soma entre " +a+ " e " +b+ " é igual a " +soma);
    }
}

File: guanabara-c/lista01/exercicio04.c
#include <stdio.h>

int main(){
    int a, b, soma;

    printf("Insira o primeiro número: ");
    scanf("%d", &a);
    printf("Insira o segundo número: ");
    scanf("%d", &b);

    soma = a + b;
    printf("A soma entre %d e %d é: %d", a, b, soma);

    return 0;
}
